### PR TITLE
Add ERC: Multi-edition NFT Distribution

### PR DESCRIPTION
**Pull request by henrywfyeung**
When opening a pull request to submit a new EIP, please use the suggested template: https://github.com/ethereum/EIPs/blob/master/eip-template.md

We have a GitHub bot that automatically merges some PRs. It will merge yours immediately if certain criteria are met:

 - The PR edits only existing draft PRs.
 - The build passes.
 - Your GitHub username or email address is listed in the 'author' header of all affected PRs, inside <triangular brackets>.
 - If matching on email address, the email address is the one publicly listed on your GitHub profile.

### CODE REVIEW

**Comment by henrywfyeung** [comment]
Could someone please advise on how we can proceed with this pull request? This is moved from the EIP repo to here recently.

File: assets/erc-7536/assets/edition.png
[the remark this answers is not in the document]
Comment (henrywfyeung) [reply assets/erc-7536/assets/edition.png:1]: Thanks I will modify accordingly

File: ERCS/erc-7536.md [hunk 1]
```diff
@@ -0,0 +1,175 @@
...
+- Limited Time: Enables minting within a particular time frame.
+
+
+## Backwards Compatibility
+
+This standard is compatible with [ERC-721](./eip-721.md).
+
+## Reference Implementation
+
+The reference implementation is given in  `../assets/erc-7536/`.
```
Comment (SamWilsn) [inline ERCS/erc-7536.md:167]:
```suggestion
The reference implementation is given in  [`../assets/erc-7536/`](../assets/eip-7536/contracts/Distributor.sol).
```

**Review by SamWilsn — COMMENTED** [review]
Looks decent. Also look at the suggestions provided by @gaudren.

File: ERCS/erc-7536.md (hunk 1 above)
Comment (henrywfyeung) [reply ERCS/erc-7536.md:167]: Sure, thanks. We will modify it according to the suggestion. At the same time, we are also revamping it to make it compatible with prior erc.

**Comment by henrywfyeung** [comment]
Still undergoing revamp. Just a message to keep this thread open

**Comment by SamWilsn** [comment]
Stale?

**Comment by henrywfyeung** [comment]
Sorry, haven't updated for a long time. We are redesigning the whole thing, and it probably will take some time. We will submit another pull request when the editing is finished.

**Pull request closed without merging**